specs/Freeze.md

```diff
@@ -0,0 +1,176 @@
+# Background
+Edge browser has a sleeping tab feature which reduces resource usage when the tab is in the background. We are introducing WebView2 APIs
+to access this feature so that invisible WebView can use less resources. We'd appreciate your feedback.
+
+
+# Description
+You may call the `TrySuspendAsync` API to have the WebView2 consume less memory. This is useful when your Win32 app becomes invisible, or when your Universal Windows Platform app is being suspended, during the suspended event handler before completing the suspended event.
```

Comment:
Asking the host to call a method seems unfortunate. Why can't the native Edge control handle going to sleep on suspension visibility changes already?

Reply:
There are website compat concern when it is suspended, and we need the app to opt-in to this behavior by calling this API.
Similarly, there is a concern for sleeping tab feature in browser, to the extend that Chrome has back off from it while Edge continue to push forward. Chrome's reasons are at https://groups.google.com/a/chromium.org/g/blink-dev/c/sotCDcI-E7Y/m/3x8clb3PEgAJ.

Reply:
Makes sense; thanks for the pointers.